CK3toEU4/Source/CK3World/Religions/Faith.cpp

```diff
@@ -39,8 +39,7 @@ void CK3::Faith::registerKeys()
 		iconPath = commonItems::singleString(theStream).getString();
 	});
 	registerKeyword("variables", [this](const std::string& unused, std::istream& theStream) {
-		const auto blobList = commonItems::blobList(theStream).getBlobs();
-		for (const auto& blob: blobList)
+		for (const auto& blob: commonItems::blobList(theStream).getBlobs())
```

Comment:
are you *sure* it's a bloblist and not another stringofItem()? All you're doing is poking it with a stick, might as well use stringOfItem for that. If it's actually a blob, then adjust the test to reflect it, wrap some variables in blobs.

Reply:
stringOfItem probably makes more sense.